conjure-go-client/httpclient/client.go

```diff
@@ -214,13 +219,13 @@ func (c *clientImpl) initializeRequestHeaders(ctx context.Context) http.Header {
 	return headers
 }
 
-func joinURIAndPath(baseURI, reqPath string) (string, error) {
+func joinURIAndPath(baseURI, reqPath string, baseURIOnly bool) (string, error) {
```

Comment:
I find this function pretty unintuitive -- can you remove `baseURIOnly` from the params here and instead use it in `doOnce` to set `b.path = """` before calling this function?